### ISSUE TITLE
particle swarm optimization option no longer working in SS_fitbiasramp

### ISSUE THREAD

**Issue by taylori** [r4ss/r4ss#174]
@James-Thorson added the option to use the particle swarm optimization option to estimate the bias adjustment ramp for recdevs in the SS_fitbiasramp function.
r4ss previously imported psoptim from the pso package, but pso seems to no longer exist.
There is a psoptim package containing a single function, psoptim, but the inputs to this function seem to be different.
@James-Thorson have you used the new version of psoptim and could you perhaps fix the SS_fitbiasramp function? I'm changing the function so it will stop with a message about this if you try altmethod='psoptim' rather than crashing.

**Comment by yukio-takeuchi** [comment]
It ssems that the package  pso is still available from CRAN (  https://cran.r-project.org/web/packages/pso/index.html ).

**Comment by taylori** [comment]
Thanks @yukio-takeuchi. You're right and I have restored the option with the commits 39d3db2 and 	b21d27b.

There was a bug in the psoptim option caused by the new option to exclude forecast years form the estimation (issue #159) and a series of silly mistakes caused me to think that the pso package had been replaced by psoptim. The bug is now fixed and the package restore to the list of r4ss dependencies.
